modules/http-api/src/main/java/org/mule/runtime/http/api/server/HttpServerConfiguration.java

```diff
@@ -27,16 +29,31 @@
   private final int connectionIdleTimeout;
   private final String name;
   private final Supplier<Scheduler> schedulerSupplier;
+  private final long readTimeout;
 
+  private static final long DEFAULT_READ_TIMEOUT_MILLIS = 30000L;
+
+  /**
+   * @deprecated since 4.5.0, use
```

Comment:
4.5.0, 4.4.1, 4.3.1? We need to discuss which is the version here.
Also, it's missing the `@since` annotation in the new methods